tests/physics/helicity_formalism/angular_distributions/epem_to_dm_d0_pip_decay/test_distributions.py

```diff
@@ -0,0 +1,160 @@
+# cspell:ignore dphi, epem
+# pylint: disable=import-outside-toplevel,redefined-outer-name
+
+import os
+from math import cos
+from typing import Any, Callable, List, Optional, Tuple
+
+import numpy as np
+import pytest
+
+
+# Use this function to reproduce the model file.
+# Note the normalization part has been removed!
+def generate_model() -> None:
+    from expertsystem.amplitude.helicity_decay import (
```

Comment:
Why inline imports?

Reply:
This function is some additional piece of info that is not required for the test. I just included it so that others understand how I created that model file that is shipped. Furthermore I wanted to make clear what is needed for running that function.... Maybe its better to just drop that function....

Reply:
Yes I think it should be clear from from the notebooks how to generate a model.